### DISCUSSION TITLE
Digital/lab5/FSM/

### DISCUSSION THREAD

**Discussion by giscus** [wintermelon008/USTC-DigitalLab-Discussion#19] [Announcements]
# Digital/lab5/FSM/

https://soc.ustc.edu.cn/Digital/lab5/FSM/

**Comment by HenRiser** [comment]
lab5 2.3一个例子中的第二段状态转移部分的case里面，S1分支的else后面多加了一个冒号

**Reply by wintermelon008** [reply]
感谢指出！已经修复啦
